AsyncDisplayKit/ASRunLoopQueue.mm

```diff
@@ -47,9 +56,17 @@ - (instancetype)initWithRunLoop:(CFRunLoopRef)runloop andHandler:(void(^)(id deq
     // the queue to stop. Attaching a custom loop source to the run loop and signal it if new work needs to be done
     CFRunLoopSourceContext *runLoopSourceContext = (CFRunLoopSourceContext *)calloc(1, sizeof(CFRunLoopSourceContext));
     runLoopSourceContext->perform = runLoopSourceCallback;
+#if ASRunLoopQueueLoggingEnabled
+    runLoopSourceContext->info = (__bridge void *)self;
```

Comment:
Does this ensure self is not retained?  Not super familiar with __bridge and the other variants of it.  Although some runloop queues are for the full app lifetime, we are planning to make new ones that may have a shorter life, such as pairing with ASDataController